DEVELOPMENT.md

```diff
@@ -9,21 +9,47 @@ The Pixie project currently consists of three main components:
 This document outlines the process for setting up the development environment for each of these components.
 
 ## Setting up the Environment
+There are two different ways to run a development environment. The first and easiest
+approach is via Docker. If you plan to use a non-Minikube environment (GKE, EKS, etc)
+for running Vizier/Pixie Cloud, use this first approach via Docker. The second approach
+is using Make and Minikube.
 
-To setup the developer environment required to start building Pixie's components, it is easiest to use our docker image which has the necessary tools and packages. If you plan to use a non-minikube environment (gke, eks, etc) for running Vizier/Pixie Cloud, this can be done by running our script:
+### Running Via Docker
+To setup the developer environment required to start building Pixie's components,
+it is easiest to run the `run_docker.sh` script. The following script will run the
+Docker container and dump you out inside the docker container console from which
+you can run all the necessary tools to build, test, and deploy Pixie.
 
-```
-./scripts/run_docker.sh
-```
+1. Since this script runs a Docker container, you must have Docker installed. To
+install it follow these instructions [here](https://docs.docker.com/get-docker/).
 
-Otherwise, to launch the dev environment with minikube, run:
+1. `run_docker.sh` requires the realpath command which is part of the coreutils
+package that you may need to install:
+    * Ubuntu: `sudo apt-get install coreutils`
+    * OS X: `brew install coreutils`
 
-```
-make dev-env-start
-```
+1. Finally, run the following script to start the Docker container:
+    ```
+    ./scripts/run_docker.sh
+    ```
+
+### Running Via Minikube
+If you plan on using a Minikube environment, launch the dev environment with Minikube,
+via Make.
+
+1. Since this runs via Minikube, you must have Minikube installed. To install it follow
```

Comment:
Can we note/link the docs.px.dev minikube setup instructions here?  The reason is, the default minikube setup will not have enough memory. https://docs.px.dev/installing-pixie/setting-up-k8s/minikube-setup/